monitoring/grafana/dashboards/tests/features/hosts_overview.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Hosts Overview Dashboard
+
+Scenario: "Test network load succeeds"
+  Given the following series:
+    | metrics | values |
```

Comment:
Is the header mandatory? Looks good to be honest, but just to know if a user can skip it...

Reply:
Behave expects headers. There a ways to do it but, it involves not using their api.

Reply:
ok!